website_header_language_switcher/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# © 2015-2019 Elico corp (www.elico-corp.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Website Header Language Switch',
+    'version': '12.0.1.0.0',
+    'category': 'website',
+    'depends': ['base', 'website'],
+    'author': 'Elico Corp,Odoo Community Association (OCA)',
```

Comment:
```suggestion
    'author': 'Elico Corp',
```
(add OCA only when pushing to OCA repo)